pragma-node/src/handlers/entries/mod.rs

```diff
@@ -77,22 +78,59 @@ pub struct GetVolatilityResponse {
 pub struct GetOnchainParams {
     pub network: Network,
     pub aggregation: Option<AggregationMode>,
-    pub timestamp: Option<u64>,
     pub routing: Option<bool>,
+    #[serde(default, with = "timestamp_serde")]
+    pub timestamp: Option<Timestamp>,
+}
+
+#[derive(Debug, Clone)]
+pub enum Timestamp {
```

Comment:
Maybe we should use this `Timestamp` type everywhere in our codebase to handle timestamps? And just restrict the range functionality for some endpoints.

Reply:
overkill imo